tests/docker-compose.yml

```diff
@@ -1,26 +1,35 @@
 version: "3"
 services:
-  centos6:
-    build: ./dockerfiles/centos6
-    cap_add:
-      - ALL
+
+  centos8:
```

Comment:
I would prefer to see 8 after 6 not before ;)

Reply:
Marking as resolved (pending on 6df0663b4097bb79cdced899c52e578f74ec0c00 approval or rejection)